crates/viewer/re_renderer/src/importer/stl.rs

```diff
@@ -20,15 +19,28 @@ pub fn load_stl_from_buffer(
 ) -> Result<CpuModel, StlImportError> {
     re_tracing::profile_function!();
 
-    let cursor = std::io::Cursor::new(buffer);
-    let StlData {
-        name,
-        triangles,
-        normals,
-        ..
-    } = StlData::read_buffer(std::io::BufReader::new(cursor)).map_err(StlImportError::TinyStl)?;
+    let mut cursor = std::io::Cursor::new(buffer);
+    let reader = stl_io::create_stl_reader(&mut cursor).map_err(StlImportError::StlIoError)?;
+    // TODO(hmeyer/stl_io#26): parse name from ASCII?
+    // https://github.com/hmeyer/stl_io/pull/26
+    let name = reader.name().cloned().unwrap_or_default();
 
-    let num_vertices = triangles.len() * 3;
+    let (normals, vertices): (Vec<_>, Vec<_>) = reader
```

Comment:
this means we're collecting the normal array twice - here and further down when assigning `vertex_normal`, that's not great given that this can be _a lot_ of. Might as well create the final one here (bytemuck cast for vertices is cheap though)

Reply:
can probably also just bytemuck down the line?